internal/injector/builtin/yaml/civisibility/testing.yml

```diff
@@ -0,0 +1,212 @@
+            fmt: fmt
+          template: |-
+            __dd_civisibility_instrumentSetErrorInfo({{ .Function.Receiver }}, "Error", fmt.Sprint({{ .Function.Argument 0 }}...), 0)
+
+  - id: Instrument testing.common.Errorf
+    join-point:
+      all-of:
+        - import-path: testing
+        - function-body:
+            function:
+              - name: Errorf
+              - receiver: '*testing.common'
+    advice:
+      - prepend-statements:
+          imports:
+            fmt: fmt
+          template: |-
+            __dd_civisibility_instrumentSetErrorInfo({{ .Function.Receiver }}, "Errorf", fmt.Sprintf({{ .Function.Argument 0 }}, {{ .Function.Argument 1 }}...), 0)
+
+  - id: Instrument testing.common.Fatal
+    join-point:
+      all-of:
+        - import-path: testing
+        - function-body:
+            function:
+              - name: Fatal
+              - receiver: '*testing.common'
+    advice:
+      - prepend-statements:
+          imports:
+            fmt: fmt
+          template: |-
+            __dd_civisibility_instrumentSetErrorInfo({{ .Function.Receiver }}, "Fatal", fmt.Sprint({{ .Function.Argument 0 }}...), 0)
+
+  - id: Instrument testing.common.Fatalf
+    join-point:
+      all-of:
+        - import-path: testing
+        - function-body:
+            function:
+              - name: Fatalf
+              - receiver: '*testing.common'
+    advice:
+      - prepend-statements:
+          imports:
+            fmt: fmt
+          template: |-
+            __dd_civisibility_instrumentSetErrorInfo({{ .Function.Receiver }}, "Fatalf", fmt.Sprintf({{ .Function.Argument 0 }}, {{ .Function.Argument 1 }}...), 0)
+
+  - id: Instrument testing.common.Skip
+    join-point:
+      all-of:
+        - import-path: testing
+        - function-body:
+            function:
+              - name: Skip
+              - receiver: '*testing.common'
+    advice:
+      - prepend-statements:
+          imports:
+            fmt: fmt
+          template: |-
+            __dd_civisibility_instrumentCloseAndSkip({{ .Function.Receiver }}, fmt.Sprint({{ .Function.Argument 0 }}...))
+
+  - id: Instrument testing.common.Skipf
+    join-point:
+      all-of:
+        - import-path: testing
+        - function-body:
+            function:
+              - name: Skipf
+              - receiver: '*testing.common'
+    advice:
+      - prepend-statements:
+          imports:
+            fmt: fmt
+          template: |-
+            __dd_civisibility_instrumentCloseAndSkip({{ .Function.Receiver }}, fmt.Sprintf({{ .Function.Argument 0 }}, {{ .Function.Argument 1 }}...))
+
+  - id: Instrument testing.common.SkipNow
+    join-point:
+      all-of:
+        - import-path: testing
+        - function-body:
+            function:
+              - name: SkipNow
+              - receiver: '*testing.common'
+    advice:
+      - prepend-statements:
+          template: |-
+            __dd_civisibility_instrumentSkipNow({{ .Function.Receiver }})
```

Comment:
Please add a newline at end of file

Reply:
done in [95dc6c3](https://github.com/DataDog/orchestrion/pull/340/commits/95dc6c315c0a7f7a4249e0fea2c57b8fd55c52ff)